framework/src/bcs/LowerDIntegratedBC.C

```diff
@@ -0,0 +1,175 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "LowerDIntegratedBC.h"
+
+#include "Assembly.h"
+#include "SubProblem.h"
+#include "SystemBase.h"
+#include "MooseVariableFE.h"
+#include "MooseVariableScalar.h"
+
+#include "libmesh/quadrature.h"
+
+InputParameters
+LowerDIntegratedBC::validParams()
+{
+  InputParameters params = IntegratedBC::validParams();
+  params.addRequiredCoupledVar("lowerd_variable", "Lagrange multiplier");
+  return params;
+}
+
+LowerDIntegratedBC::LowerDIntegratedBC(const InputParameters & parameters)
+  : IntegratedBC(parameters),
+    _lowerd_var(*getVar("lowerd_variable", 0)),
+    _lambda(_is_implicit ? _lowerd_var.slnLower() : _lowerd_var.slnLowerOld()),
+
+    _phi_lambda(_lowerd_var.phiLower()),
+    _test_lambda(_lowerd_var.phiLower())
+{
+  if (!_lowerd_var.activeSubdomains().count(Moose::BOUNDARY_SIDE_LOWERD_ID))
+    paramError("lowerd_variable",
+               "Must be defined on the subdomain BOUNDARY_SIDE_LOWERD_SUBDOMAIN that is added by "
+               "Mesh/build_all_side_lowerd_mesh=true");
+}
+
+void
+LowerDIntegratedBC::computeResidual()
+{
+  IntegratedBC::computeResidual();
+
+  prepareVectorTagLower(_assembly, _lowerd_var.number());
+
+  for (_qp = 0; _qp < _qrule->n_points(); _qp++)
+  {
+    initLowerDQpResidual();
+    for (_i = 0; _i < _test_lambda.size(); _i++)
+    {
+      _local_re(_i) += _JxW[_qp] * _coord[_qp] * computeLowerDQpResidual();
+    }
```

Comment:
```suggestion
      _local_re(_i) += _JxW[_qp] * _coord[_qp] * computeLowerDQpResidual();
```
For single instructions, let's remove surrounding curly braces